github/actions_artifacts.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// Artifact reprents a GitHub artifact.  Artifacts allow sharing
+// data between jobs in a workflow and provide storage for data
+// once a workflow is complete.
+//
+// GitHub API docs: https://developer.github.com/v3/actions/artifacts/
+type Artifact struct {
+	ID                 *int64  `json:"id,omitempty"`
+	NodeID             *string `json:"node_id,omitempty"`
+	Name               *string `json:"name,omitempty"`
+	SizeInBytes        *int64  `json:"size_in_bytes,omitempty"`
+	ArchiveDownloadURL *string `json:"archive_download_url,omitempty"`
+	Expired            *bool   `json:"expired,omitempty"`
+	CreatedAt          *string `json:"created_at,omitempty"`
+	ExpiresAt          *string `json:"expires_at,omitempty"`
+}
+
+// ArtifactList represents a list of GitHub artifacts.
+//
+// GitHub API docs: https://developer.github.com/v3/actions/artifacts/
+type ArtifactList struct {
+	TotalCount *int64      `json:"total_count,omitempty"`
+	Artifacts  []*Artifact `json:"artifacts,omitempty"`
+}
+
+// ListWorkflowRunArtifacts lists all artifacts that belong to a workflow run.
+//
+// GitHub API docs: https://developer.github.com/v3/actions/artifacts/#list-workflow-run-artifacts
+func (s *ActionsService) ListWorkflowRunArtifacts(ctx context.Context, owner, repo string, runID int64, opt *ListOptions) (*ArtifactList, *Response, error) {
```

Comment:
Can you please change `opt` to `opts`?  I know the repo is inconsistent in this regard, and it is probably mostly my fault, but I would like to standardize on `opts` because I think it makes more sense (as the type is `ListOptions`).
So while I'm maintaining the repo, I figure I can ask for what I prefer.  :smile:  Thanks.
Maybe I should file an issue to change all "opt" to "opts" to make it consistent, then I don't have to ramble on and on like this. :smile: OK, done: #1415.

Reply:
#1417 is already merged, so please change this from `opt` to `opts`.  Thank you!

Reply:
Fixed.